docs/user-guide/api-mediation/configuration-at-tls.md

```diff
@@ -14,62 +15,34 @@ Review this article for descriptions of the configuration parameters required to
     - [For communication between API Gateway and southbound services](#for-communication-between-api-gateway-and-southbound-services)
   - [Ciphers](#ciphers)
 - [Using AT-TLS for API ML in High Availability](#using-at-tls-for-api-ml-in-high-availability)
+- [Multi-tenancy deployment](#multi-tenancy-deployment)
 - [AT-TLS Troubleshooting](#at-tls-troubleshooting)
+  - [The message `This combination of port requires SSL` is thrown ](#the-message-this-combination-of-port-requires-ssl-is-thrown-)
+  - [AT-TLS rules are not applied](#at-tls-rules-are-not-applied)
+  - [Non matching ciphers](#non-matching-ciphers)
+- [Full example of AT-TLS configuration](#full-example-of-at-tls-configuration)
 
 ## AT-TLS configuration for Zowe
 
 :::tip
-Support for AT-TLS was introduced in Zowe v1.24. In this early version, startup was not possible in some versions of Zowe. For full support, we recommend that you upgrade to v2.13 or a later version of Zowe.
+Support for AT-TLS was introduced in Zowe v1.24. In this early version, startup was not possible in some versions of Zowe. For full support, we recommend that you upgrade to v2.13 or a later version.
 :::
 
 Follow these steps to configure Zowe to support AT-TLS:
 
-1. Enable the AT-TLS profile and disable the TLS application in API ML.  
-Update `zowe.yaml` with the following values under `gateway`, `discovery`, `api-catalog`, `caching-service` and `metrics-service` in the `zowe.components` section.
-
-**Example:**
+These instructions are valid for Zowe 2.18 onwards:
 
 ```yaml
 zowe:
-  components:
-    gateway:
-      spring:
-        profiles:
-          active: attls
-      server:
-        ssl:
-          enabled: false
-        internal:
-          ssl:
-            enabled: false
-    discovery:
-      spring:
-        profiles:
-          active: attls
-      server:
-        ssl:
-          enabled: false
-    api-catalog:
-      spring:
-        profiles:
-          active: attls
-      server:
-        ssl:
-          enabled: false
-    caching-service:
-      spring:
-        profiles:
-          active: attls
-      server:
-        ssl:
-          enabled: false
-    metrics-service:
-      spring:
-        profiles:
-          active: attls
-      server:
-        ssl:
-          enabled: false
+    network:
+        # For inbound traffic rules:
+        server:
+            tls:
+                attls: true
+        # If outbound traffic rules will be configured:
+        client:
+          tls:
+            attls: false
```

Comment:
why client is false? I guess it is easier to run both side (see instance configuration)

Reply:
My mistake, it should be true, although it's also true that we don't support the true value for some settings yet:
- z/OSMF http scheme
- Property in multi tenancy